components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -113,6 +116,10 @@ public AuthenticatorFlowStatus process(HttpServletRequest request,
                                            HttpServletResponse response, AuthenticationContext context)
             throws AuthenticationFailedException, LogoutFailedException {
 
+        if (isURLContainSensitiveData(request)) {
+            throw new AuthenticationFailedException("Request URL contains user credentials. Cannot send user " +
```

Comment:
what will be the error message sent out to the caller? let's make sure this message will be based all the way through.

Reply:
Fixed with https://github.com/wso2-extensions/identity-local-auth-basicauth/pull/140/commits/7b768d19dc063f94605b7091da5693496ed374da. Now the error sends back to the client with the redirectURL

<img width="494" alt="Screenshot 2022-04-22 at 5 55 54 PM" src="https://user-images.githubusercontent.com/32576163/164715886-88263af7-eaca-4b82-9338-0ebc270bedd2.png">